.github/workflows/ci.yaml

```diff
@@ -216,13 +219,13 @@ jobs:
 
       - name: Test
         run: |
-          export PATH="/usr/lib/jvm/java-11-openjdk-amd64/bin:$PATH"
+          export PATH="/usr/lib/jvm/java-1.8.0-openjdk-amd64/bin:$PATH"
```

Comment:
You used "::add-path::" above, I guess it would be good to do the same in this job for consistency.